pkg/storaged/warnings.jsx

```diff
@@ -107,5 +111,16 @@ export function find_warnings(client) {
 
     check_stratis_warnings(client, enter_warning);
 
+    for (const path in client.lvols) {
+        check_partial_lvols(client, path, enter_warning);
+    }
+
     return path_warnings;
 }
+
+export function warnings_icon(warnings) {
+    if (warnings.some(w => w.danger))
+        return <ExclamationCircleIcon className="ct-icon-times-circle" />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230912-094641-c7f396be-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)